content/topics/java-specific/functional-interface/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+content_type: topic
+ready: true
+tags:
+- functions
+title: Java Lambda expressions and Functional Interface
+---
+
+## Lambda expressions
+
+Lambda expressions, introduced in Java 8, are concise and anonymous blocks of code that can take parameters and return values, similar to methods.
+
+## Functional Interface
+
+Functional Interfaces are called Single Abstract Method (SAM) interface. They can have many default and static methods but only one abstract method - this is why they are called SAM. They make it possible to create abstractions that multiple classes can use without copying and pasting code. This is especially helpful when developers need to create a complex abstraction with various methods and behaviors.
```

Comment:
Functional Interfaces are called Single Abstract Method (SAM) interfaces. That last interfaces should be pleural.